lib/tasks/trends.rake

```diff
@@ -13,7 +13,7 @@ namespace :trends do
     current_record = 0
 
     trends.each do |trend_id|
-      puts "Working on trend_id #{trend_id}, #{(current_record += 1)} of #{trends_count}"
+      puts "Working on trend_id #{trend_id}, #{(current_record += 1)} of #{trends_count}" unless Rails.env.test?
```

Comment:
This one is ok if it's not included - it was just a way to make tests output less verbose.